src/main/java/xml/XMLreader.java

```diff
@@ -0,0 +1,124 @@
+package xml;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+/**
+ * Class for reading an xml file that was saved by the user.
+ * @author Paul
+ *
+ */
+public class XMLreader {
+
+	/**
+	 *  The document that is read from the xml file.
+	 */
+	private Document document;
+	
+	/**
+	 *  The arraylist containing the datafiles.
```

Comment:
Checkstyle vond dat er javadoc bij moest maar die rule kunnen we wel overrulen ;)